From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Use explicit version string.

Instead of using package introspection to discover the version, just use
an explicit version string.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.
---
 docs/conf.py | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index fe7aa8e..ba9fc0d 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -52,11 +52,10 @@ copyright = '2011-2022, The MoinMoin developers'
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-from pkg_resources import get_distribution
 # The full version, including alpha/beta/rc tags.
-release = get_distribution('moin').version
+release = '2.0'
 # The short X.Y version.
-version = '.'.join(release.split('.')[:2])
+version = release
 
 # The language for content autogenerated by Sphinx. Refer to documentation
 # for a list of supported languages.
